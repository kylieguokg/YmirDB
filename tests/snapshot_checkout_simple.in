snapshot
set a 28 19 0
list entries
snapshot
push a 0
set b 28
list entries
snapshot
checkout 1
list entries
set new 1 2 3
list entries
snapshot
checkout 2
list entries
checkout 3
list entries
checkout 5
checkout 0
checkout 1
list entries
checkout 4
list entries
bye
