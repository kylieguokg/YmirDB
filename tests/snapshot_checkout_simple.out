> saved as snapshot 1

> ok

> a [28 19 0]

> saved as snapshot 2

> ok

> ok

> b [28]
a [0 28 19 0]

> saved as snapshot 3

> ok

> no entries

> ok

> new [1 2 3]

> saved as snapshot 4

> ok

> a [28 19 0]

> ok

> b [28]
a [0 28 19 0]

> no such snapshot

> no such snapshot

> ok

> no entries

> ok

> new [1 2 3]

> bye
